sonic_platform_base/sonic_xcvr/sfp_optoe_base.py

```diff
@@ -21,15 +21,30 @@ def get_serial(self):
 
     def get_transceiver_info(self):
         api = self.get_xcvr_api()
-        return api.get_transceiver_info() if api is not None else None
+        result = None
+        if api is not None:
+            api.enable_cache()
```

Comment:
Will this change only apply to CMIS cables or also to other none-CMIS cables? same questions to the following changes in this file.